deploy.sh

```diff
@@ -3,9 +3,9 @@
 # Run with -h for documentation and help
 
 # testnet deployment and values (default)
-NEAR_ENV="testnet"
-oracle_account_id="fpo.opfilabs.testnet"
-provider_account_id="opfilabs.testnet"
+NEAR_ENV="testnet";
+feed_parser="switchboard-v2.testnet";
+feed_address="[251, 166, 196, 242, 159, 139, 89, 47, 230, 78, 243, 185, 185, 188, 150, 219, 165, 68, 131, 5, 216, 42, 120, 26, 26, 142, 133, 0, 111, 235, 63, 18]";
```

Comment:
Is this an [implicit account](https://docs.near.org/concepts/basics/accounts/account-id#implicit-accounts)?

Reply:
I'm not sure, but these are addresses referenced by Switchboard on their app. With the improvements I committed today, I'll stick with the string version, it's easier to manipulate.

Reply:
Okay :+1: